index.html

```diff
@@ -1,177 +1,241 @@
-                        class="counter"> Back End Dev <img src="icons/Counter.png" class="counter"> 2015</span></h3>
-            <p class="work-summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="work-used-tools">
-                <li class="tool">html</li>
-                <li class="tool">css</li>
-                <li class="tool">javascript</li>
-            </ul>
-            <button class="portfolio-btn">See Project</button>
-        </div>
-        <div class="work">
-            <div class="work-image-container work-image-3"></div>
-            <h2 class="work-name title">Tonic</h2>
-            <h3 class="small-description">CANOPY <span class="small-description-highlight"> <img src="icons/Counter.png"
-                         class="counter"> Back End Dev <img src="icons/Counter.png" class="counter"> 2015</span></h3>
-            <p class="work-summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="work-used-tools">
-                <li class="tool">html</li>
-                <li class="tool">css</li>
-                <li class="tool">javascript</li>
-            </ul>
-            <button class="portfolio-btn">See Project</button>
-        </div><div class="work">
-            <div class="work-image-container work-image-4"></div>
-            <h2 class="work-name title">Multi-Post Stories</h2>
-            <h3 class="small-description">CANOPY <span class="small-description-highlight"> <img src="icons/Counter.png"
-                        class="counter"> Back End Dev <img src="icons/Counter.png" class="counter"> 2015</span></h3>
-            <p class="work-summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="work-used-tools">
-                <li class="tool">html</li>
-                <li class="tool">css</li>
-                <li class="tool">javascript</li>
-            </ul>
-            <button class="portfolio-btn">See Project</button>
+      <div class="work" id="work-1">
+        <div class="work-image-container work-image-1"></div>
+          <div class="work-description">
+              <h2 class="work-name title">Tonic</h2>
+        	  <h3 class="small-description">
+                CANOPY
+                  <span class="small-description-highlight">
+                  <img src="icons/Counter.png" class="counter" /> Back End Dev
+                  <img src="icons/Counter.png" class="counter" /> 2015</span>
+              </h3>
+              <p class="work-summary">
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required.
+              </p>
+              <ul class="work-used-tools">
+                  <li class="tool">html</li>
+                  <li class="tool">css</li>
+                  <li class="tool">javascript</li>
+              </ul>
```

Comment:
- [ ] Kindly reduce the font-size of the technologies list in all the cards to match the Figma template. On the template, the font-size is smaller than the rest of the content of the card.
 - Your design:
 - 
<img width="825" alt="image" src="https://user-images.githubusercontent.com/61919617/214935360-71277222-1388-4c52-816f-7a0e7d4e50c9.png">

 - Figma template:
 - 
<img width="793" alt="image" src="https://user-images.githubusercontent.com/61919617/214935413-1bba1f59-1544-4353-a032-92fb341d1267.png">